source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -724,6 +729,57 @@ class NiDAQmxDriverApiTests : public Test {
     return stub()->SetScaleAttributeString(&context, request, &response);
   }
 
+  ::grpc::Status wait_for_next_sample_clock(WaitForNextSampleClockResponse& response)
+  {
+    ::grpc::ClientContext context;
+    WaitForNextSampleClockRequest request;
+    set_request_session_id(request);
+    request.set_timeout(10.0);
+    return stub()->WaitForNextSampleClock(&context, request, &response);
+  }
+
+  ::grpc::Status connect_terms(const std::string& source, const std::string& destination, ConnectTermsResponse& response)
+  {
+    ::grpc::ClientContext context;
+    ConnectTermsRequest request;
+    request.set_source_terminal(source);
+    request.set_destination_terminal(destination);
+    request.set_signal_modifiers(InvertPolarity::INVERT_POLARITY_DO_NOT_INVERT_POLARITY);
+    return stub()->ConnectTerms(&context, request, &response);
+  }
+
+  ::grpc::Status create_watchdog_timer_task_ex(double timeout, CreateWatchdogTimerTaskExResponse& response)
+  {
+    ::grpc::ClientContext context;
+    CreateWatchdogTimerTaskExRequest request;
+
+    request.set_device_name(DEVICE_NAME);
+    request.set_timeout(timeout);
+    return stub()->CreateWatchdogTimerTaskEx(&context, request, &response);
+  }
+
+  ::grpc::Status cfg_watchdog_do_expir_states(const nidevice_grpc::Session& watchdog_session, CfgWatchdogDOExpirStatesResponse& response)
+  {
+    ::grpc::ClientContext context;
+    CfgWatchdogDOExpirStatesRequest request;
+
+    set_request_session_id(request, watchdog_session);
+    request.set_channel_names("gRPCSystemTestDAQ/port1/line0:1");
+    auto expir_states = std::vector<int32>{DAQmx_Val_High, DAQmx_Val_High};
```

Comment:
Note: These should not be hardcoded, but these fields have missing enums. See AB#{1580228}